event_sourcery_esdb/event_store.py

```diff
@@ -13,6 +20,7 @@
 @dataclass(repr=False)
 class ESDBStorageStrategy(StorageStrategy):
     _client: EventStoreDBClient
+    _timeout: float = 1
```

Comment:
@Enforcer We need to think about making our implementations configurable. Not on this PR, but soon.